scripts/ci/benchmark_shared.js

```diff
@@ -2,9 +2,11 @@
 const L1_ROLLUP_CALLDATA_SIZE_IN_BYTES = "l1_rollup_calldata_size_in_bytes";
 const L1_ROLLUP_CALLDATA_GAS = "l1_rollup_calldata_gas";
 const L1_ROLLUP_EXECUTION_GAS = "l1_rollup_execution_gas";
+const L2_BLOCK_PROCESSING_TIME = "l2_block_processing_time_in_ms";
 
 // Events to track
 const ROLLUP_PUBLISHED_TO_L1 = "rollup-published-to-l1";
+const ROLLUP_BLOCK_SYNCED = "l2-block-handled";
```

Comment:
A while ago we agreed to use the name L2_BLOCK and not ROLLUP_BLOCK. I would ideally rename this to keep the naming consistent.